doc/reference_guide/source/index.rst

```diff
@@ -0,0 +1,39 @@
+.. fparser Reference Guide documentation master file.
```

Comment:
Since fparser documentation is relatively small, do we need a whole new shpinx project for this? Could this be just a page of the existing fparser shpinx?

Reply:
Yes it could and I've done that. I had a lot of trouble getting everything working so have regenerated some files.